src/common/logger.ts

```diff
@@ -19,12 +19,10 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import { app, ipcMain } from "electron";
+import { app, ipcRenderer } from "electron";
 import winston, { format } from "winston";
-import type Transport from "winston-transport";
 import { consoleFormat } from "winston-console-format";
 import { isDebugging, isTestEnv } from "./vars";
-import BrowserConsole from "winston-transport-browserconsole";
```

Comment:
Could you clarify why we are removing winston completely from renderer?

Reply:
Sure, there are several reasons:

1. Winston does not interact nicely with DevTools and the verbosity selector
2. "winston-transport-browserconsole" has a bug where any "meta-data" after the second entry is ignored (which I believe is intentional)